style.css

```diff
@@ -424,3 +433,77 @@ button[type="submit"]:hover {
 button[type="submit"]:active {
   background: #2230d2;
 }
+
+/* MEDIA QUERIES  */
+
+/* SCREENS  <=767px */
+@media screen and (max-width: 767px) {
```

Comment:
- [ ] Nice work, it is like wow! but... see this below, the card disposition should be reversed, if the first project has the image on the left and the text on the right, then the following one should have the text on the left and the image on the right, and so on

### YOUR DESIGN

<img width="701" alt="1" src="https://user-images.githubusercontent.com/77843050/228533953-c664c5b7-5504-4f3f-8b4f-579193077ca8.png">

## FIGMA DESIGN

<img width="441" alt="2" src="https://user-images.githubusercontent.com/77843050/228534243-5447f8de-133d-4404-a2ce-2f15c8c6378a.png">